akka-docs/rst/scala/stream/stages-overview.rst

```diff
@@ -587,6 +587,17 @@ when the predicate first return false and then complete.
 
 **completes** when predicate returned false or upstream completes
 
+takeUntil
```

Comment:
isn't the name confusing? compare with scala range `(1 until 5)`

If we really need this operator, should it be a parameter to `takeWhile`?